src/test/java/com/squareup/kotlinpoet/KotlinPoetTest.kt

```diff
@@ -366,7 +365,11 @@ class KotlinPoetTest {
       |import kotlin.String
       |import kotlin.Unit
       |
-      |fun ((name: String, Int, age: Long) -> Unit).whatever(): Unit = Unit
+      |fun ((
```

Comment:
Formatting here is weird due to the fact that we currently wrap parameter lists with 2 or more elements. I'll open an issue to improve it.